src/cms/views/feedback/feedbackList_view.py

```diff
@@ -0,0 +1,129 @@
+                "countUp_offer": countUp_offer,
+                "countDown_offer": countDown_offer,
+                "countUp_page": countUp_page,
+                "countDown_page": countDown_page,
+                "countUp_region": countUp_region,
+                "countDown_region": countDown_region,
+            },
+        )
+
+
+"""
+    def get(self, request):
+        feedbackList_events = EventFeedback.objects.filter(feedback__isnull=False).distinct()
+        emotions_events = {}
+        for event in feedbackList_events:
+         for item in event.feedback:
+            emotions_events[item.event]["title"] = event.title
+            if item.emotion == "POS":
+             emotions_events[item.event]["pos"] = emotions_events[item.event]["pos"] + 1
+            else:
+              emotions_events[item.event]["neg"] = emotions_events[item.event]["neg"] + 1
+
+        feedbackList_offers = OfferFeedback.objects.filter(feedback__isnull=False).distinct()
+        emotions_offers = {}
+        for offer in feedbackList_offers:
+         for item in offer.feedback:
+            emotions_offers[item.offer]["title"] = offer.title
+            if item.emotion == "POS":
+             emotions_offers[item.offer]["pos"] = emotions_offers[item.offer]["pos"] + 1
+            else:
+              emotions_offers[item.offer["neg"] = emotions_offers[item.offer]["neg"] + 1
+
+        feedbackList_regions = RegionFeedback.objects.filter(feedback__isnull=False).distinct()
+        emotions_regions = {}
+        for region  in feedbackList_regions:
+         for item in region.feedback:
+            emotions_regions[item.region]["title"] = region.title
+            if item.emotion == "POS":
+             emotions_regions[item.region]["pos"] = emotions_regions[item.region]["pos"] + 1  
+            else:
+              emotions_regions[item.region]["neg"] = emotions_regions[item.region]["neg"] + 1
+
+        feedbackList_pages = PageFeedback.objects.filter(feedback__isnull=False).distinct()
+        emotions_pages = {}
+        for page in feedbackList_pages:
+         for item in page.feedback:
+            emotions_pages[item.page]["title"] = page.title
+            if item.emotion == "POS":
+             emotions_pages[item.page]["pos"] = emotions_pages[item.page]["pos"] + 1
+            else:
+              emotions_pages[item.page]["neg"] = emotions_pages[item.page]["neg"] + 1
+
+
+        return render(
+            request,
+            self.template_name,
+            {
+                "feedbackList_events": feedbackList_events,
+                "feedbackList_offers": feedbackList_offers,
+                "feedbackList_regions": feedbackList_regions,
+                "feedbackList_pages": feedbackList_pages,
+                "emotions_events": emotions_events,
+                "emotions_events": emotions_offers,
+                "emotions_events": emotions_regions,
+                "emotions_events": emotions_pages,
+            }
+        )
+"""
```

Comment:
Please remove the file `feedbackList_view.py`. It was replaced by `admin_feedback_list_view.py` and `region_feedback_list_view.py`.